### PR TITLE
CUBICbis updates

### PR DESCRIPTION
**Pull request by maolson-msft**
Changes AIMD-friendly region algorithm to an AIMD implementation (where bytes are added to a window
value for each ACK proportional to the number of ACKed bytes) from a time-based function, to make it
easier to implement the dynamic AIMD slope that's in the new bis RFC.

### CODE REVIEW

File: src/core/cubic.c [hunk 1]
```diff
@@ -385,6 +388,10 @@ CubicCongestionControlOnDataAcknowledged(
         // Slow Start
         //
 
+        //
+        // TODO: CongestionWindow should only be allowed to grow up to SlowStartThreshold
```
Comment (maolson-msft) [inline src/core/cubic.c:392]: This is unrelated to the CUBIC bis FYI. The issue is that we intend to grow exponentially only up to SlowStartThreshold, but instead we could feasibly grow up to 2x the threshold in the slowstart code path. This is a bug that we fixed a while back in Windows TCP.

**Comment by maolson-msft** [comment]
@nibanks How do I view the WAN perf test results and compare them to the baseline?

**Comment by nibanks** [comment]
The results are in the GitHub action that merges the wanperf results. We don't currently have automated diff with main working.

**Comment by nibanks** [comment]
@maolson-msft here are this PR's results compared to main:

image

[cubicBIS.xlsx](https://github.com/microsoft/msquic/files/7841491/cubicBIS.xlsx)

**Comment by maolson-msft** [comment]
@nibanks Kind of hard to evaluate such a mass of test results, but it seems reasonable to me. A couple functional tests failed- I'm rerunning them

**Review by nibanks — APPROVED** [review]
